README.md

```diff
@@ -85,7 +86,8 @@ fi
 
 # set the node tag and wallet tag to compatible versions
 export NODE_TAG=8.1.2
-export WALLET_TAG=v2023-12-18
+export WALLET_TAG=2023.12.18
```

Comment:
This `WALLET_TAG` is needed by docker right, and is not unused?

Reply:
yes, it's needed in the docker-compose